.github/workflows/ci.yml

```diff
@@ -30,6 +30,13 @@ jobs:
         env:
           CI: true
 
+      - name: Validate JS Types
+        run: |
+          npm run build-libs
```

Comment:
There's a `build-libs` lower down in the file (because it's required for test-code), maybe it would be best to just move that higher up? File system is persisted between steps.